crates/sui-prover/tests/inputs/2.ok.move

```diff
@@ -0,0 +1,62 @@
+module 0x42::fb {
+  public fun foo() {
+    assert!(true);
+  }
+
+  public fun bar() {
+    foo();
+    assert!(true);
+  }
```

Comment:
let's make `foo` a `native` function, so that the test fails if there isn't a spec for `foo` in scope